resources/docs/database-backends.md

```diff
@@ -0,0 +1,64 @@
+# Database Backends
```

Comment:
This is already here: https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#database-backends